lib/components/rest.ts

```diff
@@ -301,6 +301,35 @@ export default function createRestAction<Context extends GatewayContext>(
             const endRequestTime = Date.now();
             requestData.requestTime = endRequestTime - startRequestTime;
 
+            const actualResponseContentType = response.headers?.['Content-Type'];
+            const expectedResponseContentType =
```

Comment:
Shouldn't this check be enabled by default? I.e. maybe default expected type should equal to `application/json` and if other type is expected it should be specified explicitly?

Reply:
In the minor version, I want to hide this option behind the setting. You can declare this option in the gateway config and verification starts work. In the next major version, we can enable it by default